pgcli/main.py

```diff
@@ -81,6 +81,10 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         self.syntax_style = c['main']['syntax_style']
         self.cli_style = c['colors']
         self.wider_completion_menu = c['main'].as_bool('wider_completion_menu')
+
+        on_error_modes = {'STOP': ON_ERROR_STOP, 'RESUME': ON_ERROR_RESUME}
```

Comment:
Nah. RAISE won't be required by main.py. RAISE is a state for pgexecute, because reraising the exception in main.py will crash everything.